data/tools/src/main/java/ee/jakarta/tck/data/tools/annp/AnnProcUtils.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package ee.jakarta.tck.data.tools.annp;
```

Comment:
Are we allowed to use this package name?

Reply:
I think so. The TCK guide points to the repo I copied this from as an example of how to run the TCK.

The build is specifically configured not to deploy this.

If I get time later I'll see how painful it would be to repackage this stuff.